webknossos/webknossos/geometry/mag.py

```diff
@@ -1,87 +1,112 @@
 import re
 from functools import total_ordering
 from math import log2
-from typing import Any, List
+from typing import Any, List, Optional, Tuple, cast
 
+import attr
 import numpy as np
 
+from .vec3_int import Vec3Int, Vec3IntLike
+
+
+def import_mag(mag_like: Any) -> Vec3Int:
```

Comment:
I'd like to mark it as private, this also ensure that it does not show up in the docs.
```suggestion
def _import_mag(mag_like: Any) -> Vec3Int:
```

Reply:
Also, can we specify `mag_like` more, type-wise? E.g.
```suggestion
MagLike = Union[Mag, Vec3IntLike, int, str]

def import_mag(mag_like: MagLike) -> Vec3Int:
```

Reply:
Unfortunately I ran into problems with attr when specifying the import parameter as MagLike (Cannot determine __init__ type from converter). MagLike can only be defined after Mag (as mag is in the Union), but the converter apparently needs a real type 🤷 If you know a trick there, let me know :)